README.md

```diff
@@ -1,25 +1,40 @@
 # PFairPlay Service
 
-## 멤버
+## 멤버 마이페이지
 
-### 1. API 기본정보
+### API 기본정보
+|메서드|요청URL|출력포맷|
+|---|---|---|
+|GET|/members/myPage|json|
 
-|메서드|요청URL|출력포맷|설명|순번|
+### 요청변수
```

Comment:
Please separate variables by whether a variable is the path variable or query param or body data.
For all of the following.